src/System.IO.FileSystem.Watcher/src/Resources/Strings.resx

```diff
@@ -127,7 +127,7 @@
     <value>Too many changes at once in directory:{0}.</value>
   </data>
   <data name="InvalidDirName" xml:space="preserve">
-    <value>The directory name {0} is invalid.</value>
+    <value>The directory name '{0}' is invalid.</value>
```

Comment:
Does this same error string appear in other projects? If it does, we should change those as well you remain consistent.

Reply:
No. Although we are pretty inconsistent with {0} vs '{0}' in general.

Reply:
> Although we are pretty inconsistent with {0} vs '{0}' in general.

For file paths? Sounds like something we should be consistent on.